packages/mui-material/src/List/List.d.ts

```diff
@@ -34,6 +36,11 @@ export interface ListTypeMap<P = {}, D extends React.ElementType = 'ul'> {
      * The system prop that allows defining system overrides as well as additional CSS styles.
      */
     sx?: SxProps<Theme>;
+    /**
+     * The variant to use. There is no variants available by default.
+     * @default 'default'
```

Comment:
I think the default variant should be `undefined`.

```suggestion
```

Reply:
Removing the default causes error in componentApiBuilder:
`variant: Error: JSDoc @default annotation not found. Add @default 'default' to the JSDoc of this prop.`

Reply:
I tried `@default undefined` also but it gave the following error:
`Error: There were errors creating prop descriptions:
  - variant: Error: Expected JSDoc @default annotation for prop 'variant' of "undefined" to equal runtime default value of "'default'"`

Reply:
Alright, I will take a look.